internal/agent/cmdchannel/runintegration/runintegration.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 New Relic Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+package runintegration
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/newrelic/infrastructure-agent/internal/agent/cmdchannel"
+	"github.com/newrelic/infrastructure-agent/internal/integrations/v4/integration"
+	"github.com/newrelic/infrastructure-agent/pkg/backend/commandapi"
+	"github.com/newrelic/infrastructure-agent/pkg/integrations/v4/config"
+	"github.com/newrelic/infrastructure-agent/pkg/log"
+)
+
+// Errors
+var (
+	NoIntNameErr = errors.New("missing required \"integration_name\"")
```

Comment:
We should agreed about Error naming convention since we use Err as prefix and suffix all over the code

Reply:
My bad, Err is better placed as prefix as seen within the stdlib variables and Error suffix for error types.

I think we should stick the this as it seems to be de-facto standard.

Reply:
https://github.com/golang/go/wiki/Errors#naming

Reply:
fixed on child PR https://github.com/newrelic/infrastructure-agent/pull/183/commits/0538a955e89f2fb60d7b38c6ef65604a1783a6b7